### PR TITLE
Fix CP Group Name in CP Objects [API-2252]

### PR DESCRIPTION
**Pull request by emreyigit**
The PR fixes wrong usage of CP groups. Before the introduced fix, the client was sending the CP group creation requests in wrong format. This was resulting that all CP objects created in `default` CP group. Now, it's fixed.

### CODE REVIEW

**Review by Patras3 — APPROVED** [review]
LGTM, thanks for the fix.

File: src/Hazelcast.Net.Tests/CP/CPRoutingTests.cs [hunk 1]
```diff
@@ -51,49 +53,63 @@ public async Task TestCPRequestRoutingToLeader()
                     config.Networking.ConnectionRetry.ClusterConnectionTimeoutMilliseconds = 10_000;
                 })).Build();
 
+
+            // Assert multiple that the addAndGet operation is sent to the leader
+            // First round creates new group
+            // Second round uses the existing group
+            for (var i = 0; i < 2; i++)
```
Comment (ihsandemir) [inline src/Hazelcast.Net.Tests/CP/CPRoutingTests.cs:60]: is this related to the change?

File: src/Hazelcast.Net.Tests/Resources/Cluster/cp.xml [hunk 2]
```diff
@@ -43,8 +43,7 @@
 
   <cp-subsystem>
     <base-dir>/custom-cp-dir</base-dir>
-    <cp-member-count>3</cp-member-count>
-    <group-size>3</group-size>
```
Comment (ihsandemir) [inline src/Hazelcast.Net.Tests/Resources/Cluster/cp.xml:47]: why removed?

File: src/Hazelcast.Net.Tests/CP/CPRoutingTests.cs (hunk 1 above)
Comment (emreyigit) [reply src/Hazelcast.Net.Tests/CP/CPRoutingTests.cs:60]: Yes.

File: src/Hazelcast.Net.Tests/Resources/Cluster/cp.xml (hunk 2 above)
Comment (emreyigit) [reply src/Hazelcast.Net.Tests/Resources/Cluster/cp.xml:47]: The default group size is already equals to cp member count at default.

**Review by ihsandemir — APPROVED** [review]
